recipes/raylib/all/conanfile.py

```diff
@@ -17,13 +18,19 @@ class RaylibConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_examples": [True, False],
```

Comment:
```suggestion
```

Official packages dont package examples and tests. The idea is provide only consumable artifacts, libraries and executables.